tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -478,6 +478,31 @@ function bar($x) {}
                     function bar($x) {}
                 ',
         ];
+
+        yield 'support for arrow function' => [
+            '<?php
+            Utils::stableSort(
+                $elements,
+                /**
+                 * @param int $a
+                 *
+                 * @return array
+                 */
+                static fn(int $a) => [$a],
+                fn($a, $b) => 1,
+            );',
+            '<?php
+            Utils::stableSort(
+                $elements,
+                /**
+                 * @param int $a
+                 *
+                 * @return array
+                 */
+                static fn($a) => [$a],
+                fn($a, $b) => 1,
+            );',
+        ];
```

Comment:
IMHO it should be:

```suggestion
        yield 'arrow function' => [
            '<?php /** @param int $a */ fn(int $a) => 1;',
            '<?php /** @param int $a */ fn($a) => 1;',
        ];

        yield 'support for arrow function' => [
            '<?php
            Utils::stableSort(
                $elements,
                /**
                 * @param int $a
                 */
                static fn(int $a) => [$a],
                fn($a, $b) => 1,
            );',
            '<?php
            Utils::stableSort(
                $elements,
                /**
                 * @param int $a
                 */
                static fn($a) => [$a],
                fn($a, $b) => 1,
            );',
        ];
```

- basic arrow function test case
- complex arrow function test case
- `@return` annotations are superfluous here, since this fixer handles only `@param`

But I was too late 😅.